packages/image_picker/image_picker_android/lib/src/messages.g.dart

```diff
@@ -0,0 +1,337 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+// Autogenerated from Pigeon (v9.1.0), do not edit directly.
+// See also: https://pub.dev/packages/pigeon
+// ignore_for_file: public_member_api_docs, non_constant_identifier_names, avoid_as, unused_import, unnecessary_parenthesis, prefer_null_aware_operators, omit_local_variable_types, unused_shown_name, unnecessary_import
+
+import 'dart:async';
+import 'dart:typed_data' show Float64List, Int32List, Int64List, Uint8List;
+
+import 'package:flutter/foundation.dart' show ReadBuffer, WriteBuffer;
+import 'package:flutter/services.dart';
+
+enum SourceCamera {
+  rear,
+  front,
+}
+
+enum SourceType {
+  camera,
+  gallery,
+}
+
+enum CacheRetrievalType {
+  image,
+  video,
+}
+
+/// Options for image selection and output.
+class ImageSelectionOptions {
+  ImageSelectionOptions({
+    this.maxWidth,
+    this.maxHeight,
+    required this.quality,
+  });
+
+  /// If set, the max width that the image should be resized to fit in.
+  double? maxWidth;
+
+  /// If set, the max height that the image should be resized to fit in.
+  double? maxHeight;
+
+  /// The quality of the output image, from 0-100.
+  ///
+  /// 100 indicates original quality.
+  int quality;
+
+  Object encode() {
+    return <Object?>[
+      maxWidth,
+      maxHeight,
+      quality,
+    ];
+  }
+
+  static ImageSelectionOptions decode(Object result) {
+    result as List<Object?>;
+    return ImageSelectionOptions(
+      maxWidth: result[0] as double?,
+      maxHeight: result[1] as double?,
+      quality: result[2]! as int,
+    );
+  }
+}
+
+/// Options for image selection and output.
```

Comment:
Nit: `video` instead of `image` since this is for the `VideoSelectionOptions`.

I was working on a separate PR and saw this.

Reply:
I'm working on `image_picker_macos`, I saw both the Android and iOS pigeons see the preferred way to pass parameters for consistency.

>  to fix it in your PR if you are already touching messages.dart though.

Should I update it in a separate PR since I'm not editing `image_picker_android`'s `messages.dart`?